map/local_ads_manager.cpp

```diff
@@ -48,6 +61,50 @@ std::chrono::steady_clock::time_point Now()
 {
   return std::chrono::steady_clock::now();
 }
+
+#ifdef TEMPORARY_LOCAL_ADS_JSON_SERIALIZATION
```

Comment:
Why do we need this ifdef? Why not just to remove this code at appropriate time?